### ISSUE TITLE
ND weather radar 

### ISSUE THREAD

**Issue by MCCVeen** [mSparks43/747-400#314]
The weather radar should be displayed exactly from the Airplane Symbol to the rose, just like the terrain map.

![image](https://user-images.githubusercontent.com/75425109/105861871-759d5600-5fef-11eb-841e-0e7310565c68.png)
![image](https://user-images.githubusercontent.com/75425109/105861918-82ba4500-5fef-11eb-925b-18236b0e9393.png)
![image](https://user-images.githubusercontent.com/75425109/105861950-8b128000-5fef-11eb-85fc-b3456730d54d.png)
![image](https://user-images.githubusercontent.com/75425109/105862061-ac736c00-5fef-11eb-9af8-12b61554ffd6.png)

**Comment by airfightergr** [comment]
Not hard to use Toto's OpernWXR. An example:
https://forums.x-plane.org/index.php?/files/file/66620-generic-rdr-4b-weather-radar/

**Comment by Matt726-S** [comment]
> Not hard to use Toto's OpernWXR. An example:
> https://forums.x-plane.org/index.php?/files/file/66620-generic-rdr-4b-weather-radar/

i don’t think so, we have users download terrain radar by drgluck, which includes a weather radar. drgluck designed his to be more realistic for boeing’s and airbuses. there are just compatibility issues that he might need to fix

**Comment by airfightergr** [comment]
Terrain Radar does not includes weather radar. Moreover with OpenWXR you can control things like tilt, gain, etc, for weather and terrain (map) mode.
